package.json

```diff
@@ -50,6 +51,7 @@
     "mocha": "^3.4.2",
     "nyc": "^11.0.3",
     "proxyquire": "^1.8.0",
-    "sinon": "^2.3.5"
+    "sinon": "^2.3.5",
+    "vinyl": "^2.1.0"
```

Comment:
`vinyl` should be in the `devDependencies`

Reply:
Thanks for your review, but `vinyl` is already in the `devDependencies`. 😄 

https://github.com/ohbarye/gulp-postcss/blob/6f15b5127f66ba7c360625c1b5e2fec4f51c575e/package.json#L47-L56

Reply:
You are right, I got confused.